linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/bash/shared.sh

```diff
@@ -4,6 +4,11 @@
 # complexity = low
 # disruption = low
 
+{{% if product in ["sle12","sle15"] %}}
```

Comment:
The rule is restricted to `sle` products. So, why the conditionals here? Would be simpler to only update the file.